modules/apps/frontend-js/frontend-js-aui-web/src/main/resources/META-INF/resources/liferay/util.js

```diff
@@ -435,6 +435,9 @@
 			return columnId;
 		},
 
+		/**
+		 * @deprecated As of Cavanaugh (7.4.x), with no direct replacement
+		 */
```

Comment:
So I am actually a little unsure about this, I think we just want to modernize these utils by removing any aui/yui dependencies, but I don't think we actually want to deprecate this utility as it is a public API for developers to consume. Unless you saw somewhere specifically about deprecating this.

And actually looking at the code for this util, I don't know if we actually have anything we need to update since there is no AUI use.

Reply:
Hey @bryceosterhaus coming at you from here now.

I don't think our deprecate vs not-deprecate debate is over. Atleast, I'm not 100% convinced after rereading the epic.

> Analyze the code's purpose and usage to determine the correct course of action: eg. deprecating (if it is no longer used), or migrating somewhere else. Remember that our goals here are two-fold: to remove all dependency on AUI so that we can turn off `frontend-js-aui-web` by default, and for any functionality that needs to live on, we want it ported into a "modernized" format (ie. remove dependency on legacy APIs and libraries, pay off technical debt etc).

And the title of the epic is `Modernize od deprecate AUI utils`

I worked on previous epics that had the same goal, and we would always deprecate utils that we didn't use, because we have so many of them and most of them are rarely used. As far as the customer is concerned, they're still there.

What you are proposing is to take this `getGeolocation`, move it to `frontend-js-web`, making a new file for it, writing tests for it? Correct? All to just cover one usage that we have, and the util is just a bad wrapper on top of a vanilla implementation.

We haven't done such things in the past when we would deprecate/modernize things, like the other utils that you can find now in `frontend-js-web`, only if it's used in more than 1-2 places in DXP we would modernize, otherwise deprecate as they're mostly bad utilities.

Reply:
@dsanz mind giving your thoughts on the matter?

Reply:
@kresimir-coko yeah I would propose that we move `getGeolocation` to `frontend-js-web` and remove any AUI in that util(if there is any), write some tests and make sure it can still be consumed exactly like it was before(no breaking changes).